common/djangoapps/student/forms.py

```diff
@@ -79,16 +80,20 @@ def save(
             send_mail(subject, email, from_email, [user.email])
 
 
+class TrueCheckbox(widgets.CheckboxInput):
+    """
+    A checkbox widget that preferentially interprets strings as false.
```

Comment:
Nit: This docstring is correct - but a more straight-forward definition would be better.